src/curtains.c

```diff
@@ -1,14 +1,181 @@
 #include "common.h"
+#include "sprite/unk_checkers.png.h"
+#include "sprite/curtains.png.h"
+#include "sprite/checkers.png.h"
 #include "ui/no_controller.png.h"
 #include "ld_addrs.h"
 #include "nu/nusys.h"
 
-extern s32 D_80076078;
-extern s32 D_800760C0;
+#include "sprite/unk_checkers.png.inc.c"
+
+Gfx D_800746E0[] = {
+    gsDPSetTextureLUT(G_TT_NONE),
+    gsSPTexture(-1, -1, 0, G_TX_RENDERTILE, G_ON),
+    gsDPLoadTextureBlock_4b(sprite_unk_checkers_png, G_IM_FMT_I, 16, 32, 0, G_TX_NOMIRROR | G_TX_WRAP,
+                            G_TX_NOMIRROR | G_TX_CLAMP, 4, 5, G_TX_NOLOD, G_TX_NOLOD),
+    gsSPEndDisplayList(),
+};
+
+#include "sprite/curtains.png.inc.c"
+
+Gfx D_80075730[] = {
+    gsDPSetTextureLUT(G_TT_NONE),
+    gsSPTexture(-1, -1, 0, G_TX_RENDERTILE, G_ON),
+    gsDPLoadTextureBlock(sprite_curtains_png, G_IM_FMT_RGBA, G_IM_SIZ_32b, 32, 32, 0, G_TX_MIRROR | G_TX_WRAP,
+                         G_TX_NOMIRROR | G_TX_CLAMP, 5, 5, G_TX_NOLOD, G_TX_NOLOD),
+    gsSPEndDisplayList(),
+};
+
+#include "sprite/checkers.png.inc.c"
+#include "sprite/checkers.pal.inc.c"
+
+Gfx D_800758A0[] = {
+    gsDPSetTextureLUT(G_TT_RGBA16),
+    gsDPLoadTLUT_pal16(0, sprite_checkers_pal),
+    gsSPTexture(-1, -1, 0, G_TX_RENDERTILE, G_ON),
+    gsDPLoadTextureBlock_4b(sprite_checkers_png, G_IM_FMT_CI, 32, 16, 0, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMIRROR |
+                            G_TX_WRAP, 5, 4, G_TX_NOLOD, G_TX_NOLOD),
+    gsSPEndDisplayList(),
+};
+
+#include "vtx/curtains1.vtx.inc.c"
+#include "vtx/curtains2.vtx.inc.c"
+#include "vtx/curtains3.vtx.inc.c"
+#include "vtx/curtains4.vtx.inc.c"
+#include "vtx/curtains5.vtx.inc.c"
+#include "vtx/curtains6.vtx.inc.c"
+#include "vtx/curtains7.vtx.inc.c"
+
+Gfx D_80075E60[] = {
+    gsDPPipeSync(),
+    gsDPSetRenderMode(G_RM_PASS, G_RM_CLD_SURF2),
+    gsDPSetCombineLERP(TEXEL0, 0, SHADE, 0, TEXEL0, 0, SHADE, 0, COMBINED, 0, PRIMITIVE, 0, 0, 0, 0, COMBINED),
+    gsSPDisplayList(D_80075730),
+    gsSPVertex(vtx_curtains7, 8, 0),
+    gsSP2Triangles(0, 1, 2, 0, 0, 2, 3, 0),
+    gsSP2Triangles(4, 5, 6, 0, 4, 6, 7, 0),
+    gsSPEndDisplayList(),
+};
+
+Gfx D_80075EA0[] = {
+    gsDPPipeSync(),
+    gsDPSetRenderMode(G_RM_PASS, G_RM_XLU_SURF2),
+    gsDPSetCombineLERP(TEXEL0, 0, SHADE, 0, 0, 0, 0, TEXEL0, COMBINED, 0, PRIMITIVE, 0, 0, 0, 0, COMBINED),
+    gsSPDisplayList(D_80075730),
+    gsSPVertex(vtx_curtains6, 31, 0),
+    gsSP2Triangles(0, 1, 2, 0, 0, 2, 3, 0),
+    gsSP2Triangles(4, 5, 6, 0, 4, 6, 7, 0),
+    gsSP2Triangles(8, 9, 10, 0, 8, 10, 11, 0),
+    gsSP2Triangles(12, 13, 14, 0, 12, 14, 15, 0),
+    gsSP2Triangles(15, 16, 17, 0, 15, 17, 18, 0),
+    gsSP2Triangles(19, 20, 21, 0, 19, 21, 22, 0),
+    gsSP2Triangles(13, 23, 24, 0, 13, 24, 14, 0),
+    gsSP2Triangles(23, 25, 26, 0, 23, 26, 24, 0),
+    gsSP2Triangles(25, 27, 28, 0, 25, 28, 26, 0),
+    gsSP2Triangles(27, 29, 30, 0, 27, 30, 28, 0),
```

Comment:
Yeah... I definitely think a `tris` or `dl` segment is in demand, here, as discussed. Editing the model would be too cumbersome otherwise. (For the future, of course.)